src/groups/mqb/mqbcmd/mqbcmd_parseutil.cpp

```diff
@@ -564,6 +570,34 @@ int parseDomainReconfigure(DomainReconfigure* reconfigure,
     return expectEnd(error, next);
 }
 
+/// ... REMOVE ...
+int parseDomainRemove(DomainRemove* remove,
+                      bsl::string*  error,
+                      WordGenerator next)
+{
+    const bslstl::StringRef domain   = next();
+    const bslstl::StringRef finalize = next();
+
+    if (domain.empty()) {
+        *error = "DOMAINS REMOVE command must be followed by a "
+                 "domain name.";
+        return -1;  // RETURN
+    }
+
+    remove->domain() = domain;
+
+    if (equalCaseless(finalize, "finalize")) {
```

Comment:
Probably `"FINALIZE"` here too, even though functionally it's the same with `equalCaseless`